pkg/storage/etcd/etcdcommon/config.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcdcommon
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"strings"
+
+	etcd "github.com/coreos/etcd/client"
```

Comment:
etcd2client

(the current etcd and etcd2 as package import names are very confusing)